extensions/amazon-dynamodb/runtime/src/main/java/io/quarkus/dynamodb/runtime/SyncHttpClientConfig.java

```diff
@@ -0,0 +1,214 @@
+package io.quarkus.dynamodb.runtime;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class SyncHttpClientConfig {
+
+    /**
+     * Type of the sync HTTP client implementation
+     *
+     * <p>
+     * Accepted values are `URL` or `APACHE`
```

Comment:
In fact, you shouldn't :). They will be automatically added to the documentation as it's an enum. So you can get rid of it.

Reply:
Thanks, I forgot about that :) Will fix.